_includes/specials/taproot/en/11-vaults-with-taproot.md

```diff
@@ -0,0 +1,71 @@
+*By [Antoine Poinsot][darosior], Revault developer* <!-- FIXME: darosior or Antoine, GitHub/Twitter/something else?  -->
+
+Bitcoin [vaults][topic vaults] are contracts playing on the finality of a payment. Numerous such protocols have been
```

Comment:
What i meant is a common characteristic to all vault proposals is the usage of a first staging transaction that timelocks the coin (Unvault) before it can be spent.

How to force the usage of this staging transaction to spenders then depends on the actual protocols:
- Onchain covenants (Moser et al)
- Presign Unvault + No way to sign another tx by having deleted the keys (Bryan Bishop's post on the ML, Jacob Swambo et al)
- Presign Unvault + Not enough keys to sign another tx (Revault)